peachjam/js/components/FindDocuments/index.vue

```diff
@@ -268,14 +268,14 @@ export default {
       },
       {
         title: this.$t('Court'),
-        name: 'court',
+        name: JSON.parse(document.querySelector('#data-labels').textContent).court,
```

Comment:
Aren't these the names used in the form submission? I don't think they should be translated, that's what `title` above is for, and the translation is done (currently) in Javascript.

Reply:
ah, because we want to facet on the different language-based fields... I see.